src/Miki/Services/StreakService.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Miki.Services
+{
+    using System;
+    using System.Threading.Tasks;
+
+    using Bot.Models;
+
+    using Framework;
+
+    using Patterns.Repositories;
+
+    public class StreakService : IStreakService
```

Comment:
There's a few issues with your service; It does not seem to elevate internal business logic to the service, but rather seem to work like a repository. I would recommend building a method that would take a user daily streak and increase it if valid. 

e.x.

```cs
public void IncreaseStreak(ulong userId) {
	var user = userService.GetUserById(userId);
	var streak = repository.Get(userId);
	if(IsValid(streak)) 
	{
		streak.value++;
 	}
	else 
	{
		streak = new DailyStreak();
	}
	repository.Edit(streak);

	transactionService.CreateTransaction(new TR { 
		Sender = 0L // Constant for the "miki bank"
		Receiver = (long)userId
		Amount = GetCurrentDailyAmount()
	});
	
	return streak;
}